packages/app/src/app/pages/Dashboard/Content/routes/Discover/Curate.tsx

```diff
@@ -204,7 +204,10 @@ export const Collection: React.FC<CollectionTypes> = ({ album }) => {
           gap={2}
           onSubmit={event => {
             event.preventDefault();
-            updateAlbum({ id: album.id, title: event.target.title.value });
+            updateAlbum({
+              id: album.id,
+              title: (event.target as any).title.value,
```

Comment:
We can type the submit event better later. Same goes for the submit event below.